### PR TITLE
[MIRROR] Suit DMI split p2: Obj icons [MDB IGNORE]

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/tgstation/tgstation/pull/69174
--------------------

## About The Pull Request

second part of #68417, which splits the suits.dmi, but this time focuses on obj items. everything stated in that pr applies to here aswell, it follows the same categorization.

## Why It's Good For The Game

finishes the much needed splitting of suits.dmi, easing the minds of spriters everywhere and lowering the probability of future pr conflicts in this area

## Changelog

no player facing changes

### CODE REVIEW

**Comment by ghost** [comment]
:( Every step of progress I make, they make two. Curse you all! Curses!!!

File: code/modules/clothing/suits/chaplainsuits.dm [hunk 1]
```diff
@@ -1,7 +1,12 @@
 //Chaplain Suit Subtypes
 //If any new staple chaplain items get added, put them in these lists
 /obj/item/clothing/suit/chaplainsuit
+<<<<<<< HEAD
 	allowed = list(/obj/item/storage/book/bible, /obj/item/nullrod, /obj/item/reagent_containers/food/drinks/bottle/holywater, /obj/item/storage/fancy/candle_box, /obj/item/candle, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
+=======
+	allowed = list(/obj/item/storage/book/bible, /obj/item/nullrod, /obj/item/reagent_containers/cup/glass/bottle/holywater, /obj/item/storage/fancy/candle_box, /obj/item/candle, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
+	icon = 'icons/obj/clothing/suits/chaplain.dmi'
+>>>>>>> dbc2435fa56 (Suit DMI split p2: Obj icons (#69174))
 	worn_icon = 'icons/mob/clothing/suits/chaplain.dmi'
```
Comment (ghost) [inline code/modules/clothing/suits/chaplainsuits.dm]:
```suggestion
	allowed = list(/obj/item/storage/book/bible, /obj/item/nullrod, /obj/item/reagent_containers/cup/glass/bottle/holywater, /obj/item/storage/fancy/candle_box, /obj/item/candle, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
	icon = 'icons/obj/clothing/suits/chaplain.dmi'
	worn_icon = 'icons/mob/clothing/suits/chaplain.dmi'
```
RECOMMEND DO NOT MERGE UNTIL https://github.com/Skyrat-SS13/Skyrat-tg/pull/15577 (drink refactor, touches the holywater in this file)

Comment (GoldenAlpharex) [reply code/modules/clothing/suits/chaplainsuits.dm]: Thanks Orion!

**Comment by GoldenAlpharex** [comment]
Let's gooooooo

**Pull request merged**